src/NLU.DevOps.ModelPerformance.Tests/models/expectedUtterances.json

```diff
@@ -71,5 +71,25 @@
     ],
     "intent": "AddAccount",
     "text": "create a new account for Bob with email bob@example.com"
+  },
+  {
+    "entities": [
+      {
+        "matchText": "two",
+        "entityValue": "csharp(1+1)"
+      }
+    ],
+    "intent": "None",
+    "text": "add two salads to my order",
+  },
+  {
+    "entities": [
+      {
+        "matchText": "tomorrow",
+        "entityValue": "csharp(DateTimeOffset.Parse(Timestamp).AddDays(1).Date.ToString(\"yyyy-MM-ddTHH:mm:ssZ\"))"
+      }
+    ],
+    "intent": "None",
+    "text": "what's on my calendar tomorrow"
   }
```

Comment:
When I run something similar to this against dialogflow. The compare is failing with "Actual utterance does not have entity matching 'tomorrow"

Reply:
I think we still have an issue with time. DF returns something like 2019-10-05T12:00:00-04:00 and what we get here is 2019-10-05T00:00:00Z so the comparison fails

---
In reply to: [331507665](https://github.com/microsoft/NLU.DevOps/pull/205#discussion_r331507665) [](ancestors = 331507665)

Reply:
Yeah good point.

Reply:
So, actually, it shouldn't be a problem. Presumably the same client that ran the DF request will also run the compare command, so it will have the same timezone. You just need to account for how DF considers the default time to be noon instead of midnight.

Reply:
I.e., your script would look like:
```
csharp(DateTimeOffset.Now.AddDays(1).AddHours(12).ToString("yyyy-MM-ddTHH:mm:sszzz")
```